pkg/cli/environment_test.go

```diff
@@ -51,9 +51,9 @@ func TestEnvSettings(t *testing.T) {
 		ns, kcontext string
 		debug        bool
 		maxhistory   int
-		kAsUser      string
-		kAsGroups    []string
-		kCaFile      string
+		asUser       string
+		asGroups     []string
+		caFile       string
```

Comment:
I saw that @bacongobbler had asked about this change in the superseded PR, so I got curious.
`revive` interprets the `k` to be Hungarian notation for a constant, which it discourages.  See https://github.com/golang/lint/issues/258

Reply:
@marckhouzam Thanks for the clarification.

Reply:
I think this is more of an issue with the linter than an issue with the code. The "k" in this case stands for "Kubernetes", as in these flags are passed down to Kubernetes client libraries.

I'd argue we should revert this particular change and silence the linter.

Reply:
From https://github.com/golang/lint/issues/258#issuecomment-260374933: "Looking at the code, it only matches on lowercase k, not uppercase K. It also doesn't match if the letter following the k isn't uppercase. So neither KBToMB nor kbToMB should get flagged." Would it be acceptable to do this as a workaround?

Reply:
@bacongobbler Replaced `k` with `kube` e.g. `kAsUser` --> `kubeAsUser`. Does this seem ok with you?